Oneirocom/Magick
# client side headers

## What Changed:
Added auth headers to missing fetch requests.
Auth headers now use a standard token regardless of local/cloud.

## How to test:

Make sure all fetch requests are working.
Events/documents/requests/fetching spells/etc. should all be working in both cloud and local

## Additional information:

N/A
